nnpdf_data/nnpdf_data/commondata/CMS_SINGLETOP_8TEV/validation/vp/runcard.yaml

```diff
@@ -0,0 +1,33 @@
+meta:
```

Comment:
Please, remove this runcard from here (maybe put in the PR for reference)

(it will be eventually removed alongside all the rawdata anyway)